students/17th/saranglee/js/login.js

```diff
@@ -0,0 +1,15 @@
+const loginContainer =  document.querySelector(".login-container");
+
+loginContainer.addEventListener("keydown", function() {
+    const loginBtn = document.querySelector(".login-btn");
+    const idInput = document.querySelector(".id-input");
+    const pwInput = document.querySelector(".pw-input");
+    
+    if(idInput.value !== "" && pwInput.value !== "") {
+        loginBtn.disabled = false;
+        loginBtn.style.opacity = "1";
+    } else if(idInput.value === "" || pwInput.value === "") {
+        loginBtn.disabled = true;
+        loginBtn.style.opacity = "0.3";
+    }
```

Comment:
- 템플릿에 질문주신 내용과 함께 말씀드릴게요! 

- value값을 변수에 지정해주지 않아, keydown이 일어날 때 값을 한템포씩 늦게 전달받아 그런것 같습니다. console.log를 사용해 idInput.value값을 한번 확인해보세요! 
- 고로, 아래와 같이 작성하여 변수에 value값이 keydown이 일어날 때마다 할당될 수 있도록 해주세요.
- 더불어, if문에 있는 조건문의 boolean값을 이용해 아래와 같이 적어주시면 좀 더 간결한 코드를 작성하실 수 있습니다. 

```suggestion
    const loginBtn = document.querySelector(".login-btn");
    const idInput = document.querySelector(".id-input").value;
    const pwInput = document.querySelector(".pw-input").value;
    
    if(idInput && pwInput) {
        loginBtn.disabled = false;
        loginBtn.style.opacity = "1";
    } else if(!idInput.value || !pwInput.value) {
        loginBtn.disabled = true;
        loginBtn.style.opacity = "0.3";
    }
```